### ISSUE TITLE
Inputs don't map to correct property when bindingPropertyName is used.

### ISSUE THREAD

**Issue by shelbyd** [gund/ng-dynamic-component#102]
### Given

```ts
class Parent {
  inputs = {
    foo: 'foo',
  };
}

@Component()
class Child {
  @Input('foo') bar: string;
}
```

### Expected

Child component has property `bar` set to `'foo'`.

### Actual

Child component has property `foo` set to `'foo'`.

**Comment by gund** [comment]
Hi, thanks for reporting this.

Yes, this use-case actually was never supported, and nobody even used it thus far =)

I have an idea how it can be fixed (using information about component's IO from it's `ComponentFactory`) but for right now do not have access to laptop so won't be able to implement a fix.

If you want to submit a PR with the fix please feel free to do so.
And if you have any questions let me know.

**Comment by gund** [comment]
Fix released in [v2.1.0](https://github.com/gund/ng-dynamic-component/releases/tag/v2.1.0)
